### ISSUE TITLE
IndexeDB tutorial example no long works because of inline playground

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#696]
```
http://www.html5rocks.com/en/tutorials/indexeddb/todo/#toc-final

See console errors. It's likely due to the inline playground trying to touch 
the DOM
that it can't access. If there's no work around, we need to move this sample 
back
to not using the inline playground. It's been broken too many times for various 
reasons.


```

Original issue reported on code.google.com by `ericbide...@html5rocks.com` on 20 Oct 2011 at 8:40

**Comment by GoogleCodeExporter** [comment]
Original comment by `han...@google.com` on 21 Oct 2011 at 6:09
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
I just see both 
http://www.html5rocks.com/en/tutorials/indexeddb/todo/#toc-final and the same 
page running on local AppEngine work on my incognito window. Could you try to 
clean the cache etc and try again?
```

Original comment by `han...@google.com` on 22 Oct 2011 at 1:27

**Comment by GoogleCodeExporter** [comment]
```
See attached. It's someone with the iframe communication.

Not sure what is going on then. People are complaining it's not working for 
them int he comments.

You're saying this doesn't produce errors for you?
http://playground.html5rocks.com/?mode=frame&hu=210&hl=150#a_simple_todo_list_us
ing_indexeddb

Chrome dev channel/canary?
```

Original comment by `ericbide...@html5rocks.com` on 22 Oct 2011 at 5:59

Attachments:
- [Screen shot 2011-10-22 at 10.57.55 AM.png](https://storage.googleapis.com/google-code-attachments/html5rocks/issue-696/comment-3/Screen shot 2011-10-22 at 10.57.55 AM.png)

**Comment by GoogleCodeExporter** [comment]
```
This issue seems to be caused by setVrequest in open() function is not 
successful.
We may need add onblocked handler of setVrequest.
```

Original comment by `han...@google.com` on 23 Oct 2011 at 5:54

**Comment by GoogleCodeExporter** [comment]
```
Fixed with revision fe54ef8d7407 and revision 84aba51e1a8e.
```

Original comment by `han...@google.com` on 23 Oct 2011 at 6:54
- Changed state: **Fixed**
